src/panels/config/info/system-health-card.ts

```diff
@@ -119,6 +159,10 @@ class SystemHealthCard extends LitElement {
         align-items: center;
         justify-content: center;
       }
+
+      #copy {
+        float: right;
+      }
```

Comment:
Make `card-header` -> `space-between`

Reply:
But then the icon is directly next to the title which suggests that you copy the title string rather than the card content unless I also add `display: flex;`.

Reply:
`space-between` only works with `display: flex` so yeah :-)